modules/react/loading-sparkles/stories/LoadingSparkles.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import {LoadingSparkles} from '@workday/canvas-kit-react/loading-sparkles';
+
+import {Basic} from './examples/Basic';
+import {RTL} from './examples/RTL';
+import {Accessible} from './examples/Accessible';
+
+<Meta title="Components/Indicators/Loading Sparkles" component={LoadingSparkles} />
+
+# Canvas Kit Loading Sparkles
+
+Loading Sparkles make users aware that an AI operation is in progress.
+
```

Comment:
Ditto as the [comment above](https://github.com/Workday/canvas-kit/pull/2497/files#diff-2c87ee8fddd0d61a3bb4dbed73103f9523c150452a1a7b6bb2ea1cc01ffa5fb8R5) – normally there's a link to the design reference here.